templates/store/snap-details.html

```diff
@@ -222,6 +222,14 @@ <h4>Users by distribution (log)</h4>
                   {% endfor %}
                 </div>
               </div>
+              {% if normalized_os|length > 10 %}
+              <div class="snapcraft-distro-chart__more u-hide">
+                <a href="/{{ package_name }}" class="js-show-more-description">Show more</a>
```

Comment:
Again for these two `data-js="show-more-description"` and `data-js="show-less-description"`